exporters/jaeger/src/main/java/io/opentelemetry/exporter/jaeger/JaegerGrpcSpanExporterBuilder.java

```diff
@@ -96,6 +96,16 @@ public JaegerGrpcSpanExporterBuilder setTrustedCertificates(byte[] trustedCertif
     return this;
   }
 
+  /**
+   * Sets the client key and chain to use for verifying servers when mTLS is enabled. The {@code
+   * byte[][]} should contain PKCS8 private key and X.509 certificate chain in PEM format.
+   */
+  public JaegerGrpcSpanExporterBuilder setClientTls(
+      byte[] privateKeyPem, byte[] privateKeyChainPem) {
```

Comment:
```suggestion
      byte[] privateKeyPem, byte[] certificatePem) {
```

Throughout

Reply:
I can rename it here. Do you want me to rename it in all places where setClientTls is used?

Reply:
Yeah

Reply:
done